src/reducers/verification.reducer.js

```diff
@@ -0,0 +1,30 @@
+import constants from '../constants/index';
+import initialState from './initialState';
+
+const { VERIFY } = constants;
+
+const verifyReducer = (state = initialState.signUp, action) => {
```

Comment:
Why not use the `verify` initial state here?

Reply:
It's because I've named my action creator "verify"

Reply:
Oh yeah, i'll change that right away